src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int INITIAL_SIZE = 16;
+    private static final int INCREASE_ARRAY_SIZE_INDEX = 2;
+
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_SIZE];
+        this.size = 0;
+        this.threshold = (int)(table.length * LOAD_FACTOR);
+    }
+
+    public static class Node<K, V> {
+        private K key;
+        private V value;
+        private Node<K, V> nextNode;
+
+        public Node(K key, V value, Node<K, V> nextNode) {
+            this.key = key;
+            this.value = value;
+            this.nextNode = nextNode;
+        }
+    }
```

Comment:
inner class should stay closer by end of mains class

Reply:
My inner class close to the end on main class. O maybe you mean that inner class have to situate  after main class methods?  I don't think so, i believe it decrease understanding of code, if inner class will be at the botton. But maybe i'm wrong and you're rigth.